front/src/applications/stdcmV2/hooks/useStdcmConsist.ts

```diff
@@ -0,0 +1,41 @@
+import { useSelector } from 'react-redux';
+
+import { useOsrdConfActions, useOsrdConfSelectors } from 'common/osrdContext';
+import { type StdcmConfSliceActions } from 'reducers/osrdconf/stdcmConf';
+import type { StdcmConfSelectors } from 'reducers/osrdconf/stdcmConf/selectors';
+import { useAppDispatch } from 'store';
+
+export default function useStdcmConsist() {
```

Comment:
We try to use `const` for hooks and components. You can either export like that and disable eslint rule or not export here and `export default` at the bottom of the file.

```suggestion
export const useStdcmConsist() {
```